sale_project_timesheet_by_seniority/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2019 Camptocamp SA
```

Comment:
Copyrights on contrary are not needed in `__init__` :) Sorry

Reply:
But as you correctly pointed above they are still present in the OCA module template so leaving them here now https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py

Reply:
Nope, copyright is not present - only license is

Reply:
Subtle difference. Changed.

Reply:
Thanks! Taking into account that it's a new module, it's better to maintain common approaches 👍